wrapper.sh

```diff
@@ -1,120 +1,114 @@
 #!/bin/sh
 
-if [ "x$DEBUG" != "x" ]
-then
+if [ "x$DEBUG" != "x" ]; then
 	set -x
 fi
 
 # The directory to store the two state files - /config is a docker standard
-CACHEFILE=/tmp/MAM.ip
-COOKIEFILE=/config/MAM.cookies
+OLD_IP_FILE=/tmp/MAM.ip
+RESPONSE_FILE=/tmp/MAM.output
+TEMP_COOKIE_FILE=/tmp/MAM.cookies
+COOKIE_FILE=/config/MAM.cookies
```

Comment:
Renamed some of these variables to use "SCREAMING_SNAKE_CASE" since I think `COOKIE_FILE` is more readable than `COOKIEFILE`.